src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -49,12 +49,24 @@ type CellRendererArgs = {
 type ColumnWidthCallback = ({ index: number }) => number;
 type GridCellData = Element<typeof FileIcon | typeof PlainButton | typeof Fragment>;
 
+type ScrollPositionClasses = {
+    'is-scrolledLeft': boolean,
```

Comment:
Are these our own class names? If so, can we prefix them with `bdl-`?

Reply:
@sahiga, Referring to the [SUIT css docs](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md#componentnameis-stateofcomponent), these are `is-stateOfComponent` category classes which are always to be used in conjunction with Component classes which are already following `bdl-` prefix. e.g. `bdl-MetadataBasedItemList-bottomRightGrid`. Please see the corresponding scss file change to see how these classes are being used.